packages/block-library/src/latest-posts/index.php

```diff
@@ -33,6 +33,9 @@ function block_core_latest_posts_get_excerpt_length() {
  *
  * @since 5.0.0
  *
+ * @global WP_Post $post Global post object.
+ * @global int $block_core_latest_posts_excerpt_length Excerpt length set by the Latest Posts core block.
```

Comment:
```suggestion
 * @global WP_Post $post                                   Global post object.
 * @global int     $block_core_latest_posts_excerpt_length Excerpt length set by the Latest Posts core block.
```

Nit: I think this isn't a strict rule, but it seems to be better to align the type, variable name, and description.

Example: https://github.com/t-hamano/wordpress-develop/blob/ed2f2ecd3fa043eca3a064a95fcb4830b5c0765c/src/wp-content/themes/twentyseventeen/inc/template-tags.php#L136-L137